src/flask_restalchemy/resources/resources.py

```diff
@@ -289,8 +297,9 @@ def get(self, relation_id, id=None):
                                                               ' Use flask-sqlalchemy and make your property return a query object')
             collection = [self._serializer.dump(item) for item in relation_list_or_query]
         else:
-            collection = query_from_request(self._resource_model, self._serializer, request,
+            query = query_from_request(self._resource_model, self._serializer, request,
```

Comment:
query_callback should be called here, but we need to discuss how it should support query from 'property resource'.